chsdi/templates/htmlpopup/windatlas50.mako

```diff
@@ -0,0 +1,738 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+  <%
+    baseUrl = request.registry.settings['api_url']
+  %>
+  <iframe src="${baseUrl}/rest/services/all/MapServer/${c['layerBodId']}/${c['id']}/extendedHtmlPopup?lang=${lang}" width="100%" height="400" frameborder="0"  style="border: 0"></iframe>
+</%def>
+
+<%def name="extended_info(c, lang)">
+
+<%
+  from chsdi.lib.raster.georaster import get_raster
+
+  r = get_raster('COMB')
```

Comment:
@gjn @danduk82 This also means that we need to keep the DTM on non-legacy instances (as it is the case at the moment)